### DISCUSSION TITLE
CHANNEL REQUEST:  COLORS TV and  SONY TV for South Asia Region

### DISCUSSION THREAD

**Discussion by MagicalHands** [iptv-org/community#1144] [Q&A]
Two channels Colors Tv and Sony TV for South Asia region. These two channels are not available and have the most watching in India.   
(Other Sony channels are present but the main one isn't)

**Comment by freearhey** [comment]
If the request is still relevant, please repost it using this [form](https://github.com/iptv-org/iptv/issues/new?assignees=&labels=channel+search&projects=&template=4_channel-search.yml&title=Find%3A+).
